src/Estimators/EstimatorManagerBase.h

```diff
@@ -265,15 +255,22 @@ class EstimatorManagerBase
   Timer MyTimer;
 
 private:
+  ///name of the primary estimator name
+  std::string main_estimator_name_;
+
   ///number of maximum data for a scalar.dat
-  int max4ascii;
+  int max4ascii_;
```

Comment:
What is the best order of member variables? Largest to smallest?

Reply:
In principle aligning the memory layout to avoid "holes" would be the ideal (that's easier with primitives, not with custom objects). I don't see the ROI in following the latter, but more on following the science, but others can chime in.